readMeWeek3

```diff
@@ -0,0 +1,73 @@
+
+a) 0…5 and 0..<5 are two types of ____________. How are they different?
+
+- 0…5 is an exclusive range — including 5.   
+- 0..<5 is a closed range and does not include 5
+
+b) Describe type inference in Swift and give an example.
+- Type inference is when Swift figures out the type by the initial value without declaring the type 
+ - var dogName = “Kobe” 
+
+c) List three advantages of Playgrounds.
+- you can run code and see instant results w/o building or running the app
+- allows you to test functions and methods w/o the need of an actual project
+- Only one file for a playground instead of multiple for Xcode project 
+
+d) When does the execution of a while loop end?
+- It ends with the condition is checked and it comes out false
+
+e) True or False: Tuples in Swift can contain values of different data types.
+- True
+
+f) List three data types you have used in Swift.
+- String
+- Int
+- Bool 
+
+g) To execute alternative code when the condition of an if statement is not met, you can use what clause?
+Else clause 
+If 2 > num {
+print( “2 is more than \(num)”)
+Else {
+print(“2 is less than \(num)
+} 
+
+h) What is the third element of the array nums defined below?  
+let nums = [5, 0, 44, 20, 1].
+-  20 is nums[3] 
+
```

Comment:
So, this is a trick question. The third element is 44, but that is store at index 2. The fourth element is store at index 3.